index.html

```diff
@@ -555,16 +555,43 @@ <h3>
         </h3>
 
         <p>
-          At least TLS 1.2 should be configured to use only strong ciphers
-          suites and to use sufficiently large key sizes. As recommendations may
-          be volatile these days, only the very latest recommendations should be
-          used. However, as a rule of thumb, the following must be used:
+          Guidance from <a
+            href="https://csrc.nist.gov/publications/detail/sp/800-52/rev-2/final">
+            NIST SP 800-52 Rev. 2
+          </a> or superceding, MUST be followed for delivery of a did web
+          document.
+        </p>
+
+        <p>
+          It is additionally recommended to adhere to OWASP's Transport Layer
+          Protection Cheat Sheet [[OWASP-TRANSPORT]] latest recommendations for
+          hardening TLS configurations.
+        </p>
+
+        <p>
+          Ephemeral keys MUST be used.
+        </p>
+
+        <p>
+          At least SHA256 MUST be used in TLS configuration, but SHA384,
+          POLY1305 or stronger is recommended, depending on the needs of your
+          operating environment.
+        </p>
+
+        <p>
+          Delete action MAY be performed by domain name registrars or DNS lookup
+          services.
+        </p>
```

Comment:
There are multiple domain name registrars, and multiple DNS lookup services.

Are we saying that any of them has the power to delete DIDs hosted on domains operated by others?

The most common reason a domain is delisted from a registrar is failure to renew services on time. In this case, does it make sense to deem all DIDs on the affected domain deleted? Once service is restored are these DIDs considered undeleted?

DNS lookup services cater to specific audiences. Preventing children from accessing adult content and preventing access to unrelated services in workplaces are common use cases for DNS filtering. What does "deleted" mean in cases where some audiences cannot access a domain, and others cannot (or, indeed, where a person can access a domain in some places or on some devices, but not others)?

Both of these examples are temporary or conditional accessibility issues. I don't think they constitute deletion.

Reply:
yeah, i generally agree - not a big fan of that section, but i kept it in (just adjusted location) for now.  if you are ok with it @gribneau i will put a separate PR together to clean up some of the operations?

Reply:
Feel free to borrow anything useful from: https://datatracker.ietf.org/doc/draft-ietf-uta-rfc6125bis/